v2/cpu.go

```diff
@@ -18,7 +18,7 @@ package v2
 
 type CPU struct {
 	Weight *uint64
-	Max    *uint64
+	Max    string
```

Comment:
@AkihiroSuda do we want to handle "max" this way or something else?  a bool value or two fields from max and period?

Reply:
I prefer having two fields

Reply:
same, these should be two separate fields

Reply:
and how do you plan to handle it in next steps? I mean writing to the file, this will be the unique case and wanted to make it in standard way.

Reply:
Guys, question I still actual, we have this func https://github.com/containerd/cgroups/blob/master/v2/manager.go#L116
I don't prefer to add exceptional case here. And since anyway we will join those two fields values in the end, I don't see a point to add two separate fields.

Reply:
Maybe like this? 
```go
type CPUMax string
func NewCPUMax(max uint64, perod *uint64) CPUMax { }
type CPU struct {
  Weight *uint64
  Max    CPUMax
  ...
}
```

Reply:
Did you me something like this? PTAL on last commit